docs/storagedrivers.md

```diff
@@ -8,10 +8,10 @@ Provided Drivers
 
 This storage driver package comes bundled with several drivers:
 
-- [inmemory](storagedriver/inmemory.md): A temporary storage driver using a local inmemory map. This exists solely for reference and testing.
-- [filesystem](storagedriver/filesystem.md): A local storage driver configured to use a directory tree in the local filesystem.
-- [s3](storagedriver/s3.md): A driver storing objects in an Amazon Simple Storage Solution (S3) bucket.
-- [azure](storagedriver/azure.md): A driver storing objects in [Microsoft Azure Blob Storage](http://azure.microsoft.com/en-us/services/storage/).
+- [inmemory](/distribution/storage-drivers/inmemory): A temporary storage driver using a local inmemory map. This exists solely for reference and testing.
```

Comment:
It looks like this is necessary for the documentation build. We can detect and pre-process these links with something like the following:

```
perl -pne 's|\[([^/].+)\]\((.+).md\)|[$1](/distribution/$2)|g' <file>
```

This takes a link with a relative markdown target and prefixes it with "/distribution/", dropping the suffix. Check the results against your diff to make sure it gets them all.